physicalplan/scan/matrix_selector.go

```diff
@@ -96,7 +96,11 @@ func (o *matrixSelector) Next(ctx context.Context) ([]model.StepVector, error) {
 		return nil, err
 	}
 
-	totalSteps := (o.maxt+o.mint)/o.step + 1
+	// Instant evaluation is executed as a range evaluation with one step.
+	totalSteps := int64(1)
+	if o.step != 0 {
+		totalSteps = (o.maxt+o.mint)/o.step + 1
```

Comment:
I guess it should be `(o.maxt-o.mint)`?

Reply:
How is this resolved?

Reply:
Sorry about that, I forgot to actually push. 😅